tutorials/howto/howto_new_profile_version.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: tutorial
+title: How to create and publish a new draft profile
+previousTutorial:
+  link: ./howto/howto_right_profile
+  title: How to select the right profile
+nextTutorial:
+  link: ./howto/howto_add_markup
+  title: How to add markup to your own resource
+
+bioschemas:
+  "@context": https://schema.org/
+  "@type": LearningResource
+  "http://purl.org/dc/terms/conformsTo":
+  - "@type": CreativeWork
+    "@id": "https://bioschemas.org/profiles/TrainingMaterial/0.9-DRAFT-2020_12_08/"
+  about:
+    - "@id": https://schema.org
+    - "@id": http://edamontology.org/topic_0089
+  audience:
+  - "@type": Audience
+    name: People interested in creating and publishing a new draft profile
+  name: "How to create and publish a new draft profile"
+  author:
+  - "@type": Person
+    name: "Leyla Garcia"
+    "@id": https://bioschemas.org/people/LeylaGarcia
+    url: https://bioschemas.org/people/LeylaGarcia
+  contributor:
+  - "@type": Person
+    name: "Alban Gaignard"
+    "@id": https://bioschemas.org/people/AlbanGaignard
+    url: https://bioschemas.org/people/AlbanGaignard   
+  dateModified: 2021-05-05
+  description: "In this how-to, we will guide you through the necessary steps for you to create and publish a new draft profile, you need some knowledge on spreadsheets, GitHub, git and Jekyll."
+  keywords: "schema.org, markup, structured data, bioschemas profile"
+  license: CC-BY 4.0
+  version: 1.0
+---
+
+## 1. Prepare your working environment
+
+To create and publish a new draft profile you need access to a couple of Bioschemas resources, here we list them all together with the main role they play in the whole process. We will explain how to use them and what to do with them in later steps.
+
+* Bioschemas specifications [GDrive folder](https://drive.google.com/drive/folders/0B8yXU9SkT3ftaWJtTGYyTTJjck0): to create the spreadsheet corresponding to the new draft profile, you need editing rights
```

Comment:
Should explain the relationship between a profile and 'specifications' or may be drop the term. Also it is unclear what the GDrive folder is and why you need it.

Reply:
Why we need (or decided to go for) a GDrive is a question out of the scope of this tutorial

Reply:
It is not about the decision to use GDrive, it is the use of the term 'specification'. that has not been defined/introduced.